pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1081,7 +1085,12 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
       throw new Error(`Can't execute commands ${ singleCommand } because there's a single-quote in them.`);
     }
     try {
-      await this.sudoExec(`/bin/sh -xec '${ singleCommand }'`);
+      if (requirePassword) {
+        await this.sudoExec(`/bin/sh -xec '${ singleCommand }'`);
+      } else {
+        await childProcess.spawnFile('sudo', ['-n', '/bin/sh', '-xec', singleCommand],
```

Comment:
Note that this might fail if the previous `sudo -n` only succeeded because it was within timeout…

Reply:
I've added `--reset-timestamp`, which should take care of this.